CHANGES.md

```diff
@@ -1,3 +1,14 @@
+## edge-18.10.4
+* CLI
```

Comment:
Can you add an extra space between the header and the first bullet here? Also, we should call out the material switchover more prominently. It's a Big Deal!!!

```
## edge-18.10.4

This releases includes a major redesign of the web frontend to make use of the
Material design system. Additional features that leverage the new design are
coming soon. This release also includes the following changes:

* CLI
  * **Fixed** Fixed an issue with the `--registry` install flag not accepting
    hosts with ports (thanks, @alenkacz!)
* Web UI
  * **New** Added a new Grafana dashboard for authorities (thanks, @alpeb!)
```